codecov.yml

```diff
@@ -6,3 +6,4 @@ coverage:
         threshold: 0.01%  # the leniency in hitting the target
   ignore:
   - "src/**/visualization/**/*"
+  - "src/models/differentiable/R/rtorch_model.jl"
```

Comment:
Do you think there's any way to add the loss and loader functions into the ignore section as well? Since they're not testable for now, it shouldn't be something Codecov looks at in my opinion.